src/app.rs

```diff
@@ -521,6 +539,7 @@ pub fn app() -> App<'static, 'static> {
         .subcommand(cmd_property_names)
         .subcommand(cmd_property_values)
         .subcommand(cmd_case_folding_simple)
+        .subcommand(cmd_case_mapping)
```

Comment:
Is there a particular reason why you chose `case-mapping` for this instead of perhaps `case-folding`, so as to be consistent with `case-folding-simple`? Moreover, is the `--simple` flag necessary given the existence of `case-folding-simple`?

Reply:
Ah I see now. `case-folding-simple` is really about case _folding_ as opposed to case _conversion_. Would it perhaps be better to call this `case-conversion` instead of `case-mapping`? I guess "case mapping" is used in the Unicode spec, so that might be the better name after all.

Reply:
Yeah, case folding and case mapping are different. Case folding is mostly for performing case insensitive comparison -- You're not really supposed to show the user the result of case folding, since I gather it can look weird for some languages (and will often be distinct from the output of any case mapping).

I can change the name if you want. I think mapping is the right term but I sympathize with the confusion.

Reply:
Nah it's all good. I think my suggested edit below to the CLI docs would clear it up!